Formula/rotz.rb

```diff
@@ -0,0 +1,18 @@
+class Rotz < Formula
+  desc "Fully cross platform dotfile manager and dev environment bootstrapper written in Rust."
+  homepage "https://volllly.github.io/rotz/"
+  url "https://github.com/volllly/rotz/archive/refs/tags/v0.9.3.tar.gz"
+  sha256 "b7556bdfbee63c9def84a6ce8b12fa41d3f7ee53e2d3b3f240d35110c9ed2f1b"
+  license "MIT"
+  head "https://github.com/volllly/rotz.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--features", "all-formats", *std_cargo_args
+  end
+
+  test do
+    assert_match "rotz ${version}", shell_output("#{bin}/rotz -v")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.